src/libs/actions/Report.js

```diff
@@ -432,6 +432,67 @@ function unsubscribeFromReportChannel(reportID) {
     Pusher.unsubscribe(pusherChannelName);
 }
 
+/**
+ * Get the report ID, and then the actions, for a chat report for a specific
+ * set of participants
+ *
+ * @param {String[]} participants
+ */
+function fetchOrCreateChatReport(participants) {
+    let reportID;
+
+    if (participants.length < 2) {
+        throw new Error('fetchOrCreateChatReport() must have at least two participants');
+    }
+
+    API.CreateChatReport({
+        emailList: participants.join(','),
+    })
+
+        .then((data) => {
+            if (data.jsonCode !== 200) {
+                alert(data.message);
+                return;
+            }
+
+            // Set aside the reportID in a local variable so it can be accessed in the rest of the chain
+            reportID = data.reportID;
+
+            // Make a request to get all the information about the report
+            return API.Get({
+                returnValueList: 'reportStuff',
+                reportIDList: reportID,
+                shouldLoadOptionalKeys: true,
+            });
+        })
+
+        // Put the report object into Onyx
+        .then((data) => {
+            if (data.reports.length === 0) {
```

Comment:
For the non-200 case, `data` here will be `null` and you can't access properties of something that is `null` so JS is going to throw a fatal error here.

Reply:
Removed this part from the function.